### DISCUSSION TITLE
Use in a banking site

### DISCUSSION THREAD

**Discussion by jccazeaux** [add2cal/add-to-calendar-button#599] [Q&A] [answered]
Hi, we would like to use your opensource solution in our banking website. The customer can create a meeting with the bank, we would like to add the button so the customer can add the created event in his calendar. Is it possible with the opensource license?

Thank you

**Comment by jekuer** [comment, marked as answer]
Hi @jccazeaux ,

Yes, this is possible as you are using it on your own website (assuming you represent the bank) and not as some kind of manages solution or product.
Mind that it would not be allowed if you are building a SaaS for banks offering this service!

I would also recommend you to have a look at our PRO offering. Since it comes with server-side support, it is more stable and you could generate ics files on the fly to add to confirmation emails.
See https://docs.add-to-calendar-pro.com/recipes/dynamic-checkout.html for that specific case and https://add-to-calendar-pro.com for general information on that.
